packages/invoices/src/InboundShipment/ListView/api.ts

```diff
@@ -86,15 +63,8 @@ export const getInboundShipmentListViewApi = (
     };
     return async (): Promise<{ nodes: InvoiceRow[]; totalCount: number }> => {
       const result = await api.invoices(queryParams);
-
-      const invoices = invoicesGuard(result);
-
-      const nodes = invoices.nodes.map(invoice => ({
-        ...invoice,
-        pricing: pricingGuard(invoice.pricing),
-      }));
-
-      return { nodes, totalCount: invoices.totalCount };
+      const invoices = result.invoices;
```

Comment:
not 
```
return result.invoices;
```

or

```
const { invoices } = result;
return invoices;
```
? I feel that Will would want me to ask about the destructuring 😉

Reply:
The legacy of william thomson! Thanks 😁